platform/broadcom/sonic-platform-modules-juniper/debian/files

```diff
@@ -0,0 +1,3 @@
+sonic-juniper-platform-modules_1.1_amd64.buildinfo main extra
+sonic-platform-juniper-qfx5200_1.1_amd64.deb main extra
+sonic-platform-juniper-qfx5210_1.1_amd64.deb main extra
```

Comment:
this looks like generated file, please remove.

Reply:
@lguohan Done.